src/main/webapp/dashboard.js

```diff
@@ -27,44 +27,59 @@ import {getUser} from './user.js';
  */
 
 // ELEMENTS_FOR_ABOUT_ME_SECTION
+/**
+ * Ids of elements from dashboard.html.
+ * @type {string}
+ */
+const USER_BIO_ID = 'user-bio';
+const USER_LOCATION_ID = 'user-id';
+const USER_NAME_LOCATION_ID = 'user-name-location';
+export const ABOUT_ME_CARD_ID = 'about-me-card-body';
+export const USER_EMAIL_ID = 'user-email';
+export const USER_GITHUB_ID = 'user-github';
+export const USER_IMAGE_ID = 'user-image';
+
 /**
  * Elements from dashboard.html.
  * @type {HTMLElement}
  */
-const aboutMeCardDiv = document.getElementById('about-me-card-body');
-const userBioElement = document.getElementById('user-bio');
-const userImageElement = document.getElementById('user-image');
-const userGithubButton = document.getElementById('user-github');
-const userEmailButton = document.getElementById('user-email');
 
 /**
  * Populate the card element 'aboutMeCardDiv' with information about a
  * given user.
  * @param {User} user
+ * @param {Document} document - The HTML document to be this code will run on.
+ * This decision will help simplify testing.
  */
-function createAboutMe(user) {
+export function createAboutMe(user, document) {
+  const aboutMeCardDiv = document.getElementById(ABOUT_ME_CARD_ID);
+
   const editButton = createEditButton('#');
   aboutMeCardDiv.prepend(editButton);
 
+  const userImageElement = document.getElementById(USER_IMAGE_ID);
   userImageElement.src = user.image;
 
   const userNameAndLocationElement =
-    document.getElementById('user-name-location');
+    document.getElementById(USER_NAME_LOCATION_ID);
   userNameAndLocationElement.innerHTML = user.name + '<br>';
   if (user.location) {
     const userLocation = createLocation(user.location);
     userNameAndLocationElement.append(userLocation);
   }
 
+  const userGithubButton = document.getElementById(USER_GITHUB_ID);
   userGithubButton.href = user.gitHubURL;
+  const userEmailButton = document.getElementById(USER_EMAIL_ID);
   if (!user.email) {
```

Comment:
Nit Q: Could a user ever have access to the dashboard page and not have an email?

Reply:
No, I don't think so. I added this in case we would make this into a reusable breakout page. But we recently went in a different direction so I'll remove the email and GitHubURL stuff